jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Connection.java

```diff
@@ -56,6 +56,8 @@
     private final ISession session;
     private final int bufferSize;
     private final ExecutionStrategy strategy;
+    private boolean useInputDirectBuffers;
```

Comment:
use `fooBarDirectBuffers` or `fooBarDirectByteBuffers` everywhere, but not both!